### ISSUE TITLE
`meson install` raises AttributeError with emscripten

### ISSUE THREAD

**Issue by Akaricchi** [mesonbuild/meson#7542]
**Describe the bug**
```
$ meson install
ninja: Entering directory `.'
[2/5] Generating version information with a custom command
Installing resources/bundle_gfx.data to /data/webshit/ports/taisei/out/.
Traceback (most recent call last):
  File "/data/git/meson/mesonbuild/mesonmain.py", line 131, in run
    return options.run_func(options)
  File "/data/git/meson/mesonbuild/minstall.py", line 542, in run
    installer.do_install(datafilename)
  File "/data/git/meson/mesonbuild/minstall.py", line 362, in do_install
    self.install_targets(d)
  File "/data/git/meson/mesonbuild/minstall.py", line 515, in install_targets
    depfixer.fix_rpath(outname, t.rpath_dirs_to_remove, install_rpath, final_path,
AttributeError: 'TargetInstallData' object has no attribute 'rpath_dirs_to_remove'
```

Note that Emscripten has no concept of `rpath`.

**To Reproduce**
Unfortunately, I have not been able to reproduce this with anything less complicated than [Taisei](https://github.com/taisei-project/taisei) so far.

**Expected behavior**
Successful installation.

**system parameters**
* Cross build (emscripten)
* openSUSE Tumbleweed
* Python 3.8.4
* meson 0.55.999 (6006987ce)
* ninja 1.10.0

**Comment by nirbheek** [comment]
Can you reproduce this with a fresh builddir too?

**Comment by Akaricchi** [comment]
I did run --wipe on it, so it should be as good as fresh? I'll try a completely new build dir a bit later just to make sure, though.

**Comment by jpakkane** [comment]
It should. But looking at the code I can't understand how that could fail other than some sort of weird pickling error. Please try a fresh build, just to be sure.

**Comment by Akaricchi** [comment]
Ok, I've just finally tried it with a completely clean build dir, and the issue is gone. I've retried it with `--wipe` on the old build dir just to be sure as well, and I can't reproduce it there anymore, either. I must've forgotten to `--wipe` before after all, although I was convinced that I did... Shocking that it even built fine, because that build dir hasn't been touched in a while.

Anyway, this is resolved now; not a bug. Apologies for wasting everyone's time.
